types/src/content/index.ts

```diff
@@ -19,7 +19,7 @@ export class CurrencyAmount extends CurrencyOf {}
 
 // NFT types
 
-export class Royalty extends UInt {}
+export class Royalty extends UInt.with(32, 'Perbill') {}
```

Comment:
What does `32` here mean? It will be a 32 bit value?

Reply:
Yes